src/igvfd/tests/test_types_in_vitro_system.py

```diff
@@ -113,3 +113,16 @@ def test_summary(testapp, in_vitro_cell_line, in_vitro_differentiated_cell, huma
     res = testapp.get(in_vitro_differentiated_cell['@id'])
     assert res.json.get(
         'summary') == 'lymphoblastoid differentiated cell specimen line induced to brown adipose tissue for 5 minutes, Homo sapiens transfected with a reporter library (MOI of 2), grown in DMEM with serum'
+    testapp.patch_json(
+        in_vitro_cell_line['@id'],
+        {
+            'time_post_change': 5,
```

Comment:
Can the `classifications` also be patched to be "gastruloid"? I think this fixture is originally "cell line".

Reply:
ahh i missed that. fixed.